framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -315,6 +316,10 @@
     // org.checkerframework.framework.util.typeinference.DefaultTypeArgumentInference
     "showInferenceSteps",
 
+    // Output stack trace when checker encounters errors
```

Comment:
Use the short message I proposed for the manual.

Reply:
I have used the short message in my latest commit.